source/reporting/index.html.md.erb

```diff
@@ -267,15 +269,17 @@ Example response:
 
 Where:
 
-- `total_amount` is the amount your user paid in pence, including a [corporate card surcharge](/corporate_card_surcharges/#add-corporate-card-fees) if you added one
+- `total_amount` is the amount your user paid in pence and only appears if a [corporate card surcharge](/corporate_card_surcharges/#add-corporate-card-fees) was added to the payment
 - `fee` is the PSP transaction fee that we took from the amount your user paid, in pence
 - `net_amount` is the amount that will be paid into your account, in pence
 
-The response will not contain a `fee` or `net_amount` parameter if either:
+The response will not contain `fee` or `net_amount` if either:
 
 - your PSP is Worldpay - we do not deduct these PSPs' transaction fees
 - you're using an API key from your test ('sandbox') account - we do not deduct fees from payments in this account
 
+Stripe may charge transaction fees on failed payments. If this happens, `net_amount` will be a negative number. From 1 April 2022, this negative amount will be taken from your Stripe account balance. For example, a `net_amount` of `-200` will take £2.00 from your Stripe account.
```

Comment:
One point for clarification. With out current implementation plan, I don't think we're going to show the fees for failed payments in the API until it's also taken from the Stripe balance, on 1st April. Before then, people will only be able to see these fees for failed payments on their test Stripe account.

Reply:
@stephencdaly Is this clearer?

> Stripe may charge transaction fees on failed payments. Until 1 April 2022, GOV.UK Pay will cover the cost of transaction fees on failed payments and these payments will not appear in a `GET /v1/payments/{PAYMENT_ID}` call.

> If Stripe charge transaction fees on a failed payment on or after 1 April 2022, `net_amount` will be a negative number. From 1 April 2022, this negative amount will be taken from your Stripe account balance. For example, a `net_amount` of `-200` will take £2.00 from your Stripe account.

Reply:
Yep I think that's very clear. Do we need to detail specifically that we're covering the cost, or just that the fees will appear and be deducted from the balance after 1st April? That's probably a question for Sacha.

Reply:
I asked Sacha about it and he wants it included. We'll take it out in the first week of April.

Thanks for taking a look!